### PR TITLE
Bf/config log level

### LINKED ISSUE #253: 2.7.0-dev (de)Bug?: Config Log-Level ineffectual.
For whatever reason, if I edit the config from `Debug` to `Info`, and the `debug` logs still seem to keep coming.  Not sure if this is `2.7.0` exclusive or a previous issue, but restarting the service, and the pi, do not resolve this.

Additionally, as I have been developing, I have removed and installed completely new builds several times, and the debug logs are still coming...

### PR DESCRIPTION
**Pull request by thebeline**
Potentially resolves #253 .

I am not sure WHY this was an issue, but...  We literally never used the configured log level...  Also, this single line I changed...  It just read the current log level and set it back...  I feel like I am missing something...

### CODE REVIEW

**Comment by thebeline** [comment]
Can confirm, works a treat.  Much betterer

File: main.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func main() {
 
 
 func setLogLevel() {
-	logLevel := logger.LogLevel()
+	logLevel := strings.ToLower(os.Getenv(utils.EnvLogLevel))
```
Comment (JeffB42) [inline main.go:148]: This is essentially what logger.LogLevel() does.  SetLogLevel() and LogLevel() are:
```
func SetLogLevel(newLevel logrus.Level) {
	_logLevel = newLevel
	_strLogLevel = strings.ToLower(_logLevel.String())

	_logrusLogger.SetLevel(_logLevel)
	standardLog.Printf("logger.SetLogLevel() - the log level is now set to: %s", _strLogLevel)
}

func LogLevel() string {
	// Returns a lower case string.
	return _strLogLevel
}
```

Initially, LogLevel() was named "LowerCaseLogLevel()" but that seemed too wordy to me.  I haven't verified it yet, but my guess is that the cached value for _strLogLevel is getting set (probably defaulting) to debug, and then never getting set from the config file.  Really good find, thanks!

A better solution is to fix the bug at the root cause (in this case `SetLogLevel()` and `LogLevel()`) and not patch it here in `main.setLogLevel()`  Patching it here means that logLevel is the correct value, but anything calling logger.logLevel() still has the bug.

Comment (thebeline) [reply main.go:148]: The lowercase is for reading the User Input.

Please review the logger source, and indeed the application source, as mentioned in #253 the Configured level was never read in at all.  And `logger` just set a default to `debug`.

This only lower-cases to prepare for the switch.  This is the correct place for the patch.

Comment (JeffB42) [reply main.go:148]: yep.  I think this was a case where I was in the middle of updating & correcting my reply, and you replied at the same time.

Comment (thebeline) [reply main.go:148]: LOL.  LOVE IT.  Serendipitous.

WHat are you even doing working on this on a Monday night, anyway.  I thought you were a weekend guy?  :-D

Comment (JeffB42) [reply main.go:148]: I was only looking at the single line/diff in your PR, and it initially looked wrong, but I just checked the code and looked at the flow, and yep, your PR is correct.  I'm merging it in (and also signing off for the night)

Comment (JeffB42) [reply main.go:148]: > LOL. LOVE IT. Serendipitous.
> 
> WHat are you even doing working on this on a Monday night, anyway. I thought you were a weekend guy? :-D

there's like, 20 messages in my inbox from this guy called "thebee" whose been posting PRs...  ;-)  LOL

**Review by JeffB42 — APPROVED** [review]
